src/index.js

```diff
@@ -8,7 +8,7 @@ let pageNum = 1;
 const pokemonsPerPage = 12;
 const pokeHeader = document.querySelector('.modal-header');
 const pokeDetails = document.querySelectorAll('.details');
-const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const closeModalButtons = document.querySelectorAll('.close-button');
```

Comment:
That was previously corrected, isn't it? We need to double-check the changes in the merge.